src/Workspaces/Remote/ServiceHub/Services/DesignerAttribute/RemoteDesignerAttributeService.cs

```diff
@@ -33,7 +33,7 @@ public Task StartScanningForDesignerAttributesAsync(CancellationToken cancellati
                     analyzerProvider,
                     new IncrementalAnalyzerProviderMetadata(
                         nameof(RemoteDesignerAttributeIncrementalAnalyzerProvider),
-                        highPriorityForActiveFile: false,
+                        highPriorityForActiveFile: true,
```

Comment:
This change is no longer relevant right?

Reply:
it still is for me, eve if it's not respected.  i.e. if our OOP crawler can support this concept at some point, we would want this prioritization for these files.  It will mean that changes to them are realized in the UI at least a second faster, which would be preferred over the current approach. i.e. we'd likely get down to about 1-1.5 seconds from teh current 2-2.5 seconds.